### DISCUSSION TITLE
Void Sprites are Real!

### DISCUSSION THREAD

**Discussion by xX-Dillinger-Xx** [endless-sky/endless-sky#9927] [General]
I don't know who knows this, but void sprites are real. However, they don't live in space. 

[(video) Blue Dragon Sea Slugs](https://www.theweathernetwork.com/en/news/nature/animals/beware-the-adorable-but-dangerous-blue-dragon-sea-slug-this-spring)

**Comment by Zitchas** [comment]
Yes, yes they are. Cute little things.

**Reply by xX-Dillinger-Xx** [reply]
@Zitchas 
Who was the one who decided to use them as inspiration for the void sprites?

**Reply by Zitchas** [reply]
That was, I believe, MZ. The Creator of Endless Sky.

**Reply by xX-Dillinger-Xx** [reply]
I almost spit my coffee all over place, when I first saw that video pop up. I've never heard of them before. I thought I was about to enter the twilight zone. 
You know you play too much when ...................

**Reply by Zitchas** [reply]
Yeah, neat little things. It's worth taking a browse through libraries of sea slug videos, honestly. There's a lot of stuff in the oceans that really looks like alien creatures.

![02TB-SEASLUG-jumbo](https://github.com/endless-sky/endless-sky/assets/32169904/e41ce72e-acf0-418c-b5e9-aa12d96d0492)

I'd love to get this into the Ember Waste sometime... Good luck modelling that, though.

**Reply by xX-Dillinger-Xx** [reply]
>I'd love to get this into the Ember Waste sometime

Then do it. It already has all the glowing bits and everything. 

>Good luck modelling that, though.

Put it out to the community, somebody will tackle it.
